src/content/refi/index.md

```diff
@@ -0,0 +1,115 @@
+
+2. Incentives are misaligned, as the desire to accumulate more wealth and the actions following this desire, which is a strong driver in capitalistic systems, often negatively impact our environment.
+
+These factors typically lead to all available resources being consumed, with no regard for negative impacts on the environment or society at large, resulting in tragedies like polluted air and water, deforestation, or exhaustion of soils and fisheries - but such situations are not intractable. Traditional solutions to public goods problems like the Tragedy of the Commons typically involve top-down solutions enforced by a government or other institution with enforcement power to prevent excessive extraction from such Common Pool Resources (CPRs) as fisheries, watersheds, and the atmosphere we all breathe.
+
+Research on systemic solutions to the Tragedy of the Commons was pioneered by [Elinor Ostrom](https://en.wikipedia.org/wiki/Elinor_Ostrom#Research), and implementing regenerative solutions to these public goods coordination failures is the defining purpose of ReFi. Everyone benefits from clean air and water, a stable climate, and conservation of natural beauty and biodiversity, but they are also driven by the desire to generate income to sustain (and improve) their own lives. We must therefore develop mechanisms that align our need to “manage the commons” with status quo incentive structures that reward exploitation of Common Pool Resources. A key example of such a regenerative system that incentivizes planet-positive activity is [carbon credit markets](https://www.klimadao.finance/de/blog/cop26-establishes-rules-for-carbon-markets).
+
+## Carbon credits and carbon markets {#carbon-credits}
+
+[Carbon credits](https://www.klimadao.finance/de/blog/the-role-of-carbon-mitigation-and-removal-credits) and their associated markets, specifically the voluntary carbon market (VCM), are currently the [most well-known use case of ReFi](https://www.klimadao.finance/blog/klimadao-a-catalyst-for-innovation-within-the-voluntary-carbon-market). They address both of the issues mentioned above.
+
+- Carbon credits facilitate an economic choice that, before they existed, was not possible. Landowners, for example, may use a forest’s resources to create a monetary return by selling or using the wood it supplies. As an alternative, carbon credits offer the chance to leave the forest unharmed while the landowner is paid to leave it alone. The carbon credits generated by this forestry project are in high demand by companies who wish to offset their carbon footprints. The landowner gets paid for his or her work, the company keeps producing its products and services, and the forestry project sequesters the carbon emitted by the company’s operations. This regenerative flow, which sits in contrast to extractive, exploitive finance, is the core of ReFi.

```

Comment:
Fair enough. Out of scope for this page but I'm interested in your take on the criticism that ReFi doesn't do anything to solve fundamental problems with C credit systems - e.g. replacing C in rocks with C in wood is not a like-for-like trade. It takes "cold storage" C and replaces it with fast "hot wallet" C.

Reply:
Would be happy to discuss this in more depth, maybe on a call

I think your specific question is about nature-based carbon removal such as afforestation/reforestation (a.k.a. "planting trees"?) where carbon dioxide that was released from burning fossil fuels is sequestered in trees which only live for at best a few centuries.

Afforestation/reforestation is just one of many methodologies for carbon removal, and there are also many methodologies for _avoided_ emissions which incentivize adoption of renewable energy and protection of existing forests from deforestation, which is arguably more effective per $ spent than removing emissions after-the-fact.

In my opinion we need a wide array of methodologies, some of which have greater permanence like geological storage or avoiding emissions in the first place by switching to renewables, and others which have greater cobenefits (like restoring natural ecosystems or improving water quality). See this article on permanence for more details: https://www.offsetguide.org/high-quality-offsets/permanence/

The carbon markets are a complex space, and there's a lot of nuance and shades of gray. No methodology is perfect, they all make assumptions and it's up to scientific experts to evaluate each proposed methodology before they are accepted into a standard like Verra or Gold Standard, or a web3-native registry system like Regen Network is building.